### PR TITLE
test(cypress): add feature files

### PR DESCRIPTION
**Pull request by ismay**
https://jira.dhis2.org/browse/DHIS2-10472

### CODE REVIEW

**Review by Mohammer5 — CHANGES_REQUESTED** [review]

File: cypress/integration/add-route/create-parameter-jobs.feature [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    Scenario Outline: User creates a <job-type> job
+        Given the user navigated to the add job page
+        And the user enters a job name
+        And the user selects the <job-type> job type
+        And the user enters a <schedule-type> schedule
+        And the user enters the parameters for <job-type>
+        When the user clicks the save button
+        Then the <job-type> job data is sent to the backend
+
+    Scenarios:
```
[the remark this answers is not in the document]
Comment (ismay) [reply cypress/integration/add-route/create-parameter-jobs.feature:12]: I thought Scenarios and Examples were synonyms. Is there a difference I'm not aware of maybe? Here's the section from the Gherkin docs:

Screenshot 2021-02-17 at 14 38 00

https://cucumber.io/docs/gherkin/reference/#example

File: cypress/integration/add-route/create-parameter-jobs.feature [hunk 2]
```diff
@@ -0,0 +1,22 @@
+Feature: Users should be able to create jobs that take parameters
+
+    Scenario Outline: User creates a <job-type> job
```
[the remark this answers is not in the document]
Comment (ismay) [reply cypress/integration/add-route/create-parameter-jobs.feature:3]: I based this on this post: https://www.baeldung.com/cucumber-scenario-outline (it's the first hit after the Gherkin docs, when I was searching for some scenario outline examples). I haven't tested this yet, but my reason for doing this was that it seemed more readable for the test results (that way you'll know which job type is being run).

**Comment by ismay** [comment]
@Mohammer5 Left a couple comments explaining why I chose the syntax used in these features. Let me know what you think, and if that makes sense!

File: cypress/integration/list-route/job-toggle.feature [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        Then the on/off switch is <after>
+
+    Scenarios:
+        | owner |   action |  initial | before | after |
+        |  user | disables |  enabled |     on |   off |
+        |  user |  enables | disabled |    off |    on |
+
+    Background:
+        Given the user navigated to the job list page
+        And system jobs are visible
```
[the remark this answers is not in the document]
Comment (ismay) [reply cypress/integration/list-route/job-toggle.feature:19]: Ah ok I didn't know that. I'll update it 👍

[the remark this answers is not in the document]
Comment (ismay) [reply cypress/integration/list-route/job-toggle.feature]: Fixed in ddcad7b

File: cypress/integration/add-route/create-parameter-jobs.feature (hunk 1 above)
[the remark this answers is not in the document]
Comment (Mohammer5) [reply cypress/integration/add-route/create-parameter-jobs.feature:12]: Ah, interesting, was not aware! I'd prefer `Examples` as that's visually more distinct to; `Scenario`, but as it's working this way, I suppose it doesn't really matter

File: cypress/integration/add-route/create-parameter-jobs.feature (hunk 2 above)
[the remark this answers is not in the document]
Comment (Mohammer5) [reply cypress/integration/add-route/create-parameter-jobs.feature:3]: Ah very interesting!

**Review by Mohammer5 — APPROVED** [review]

**Comment by ismay** [comment]
Thanks for the review @Mohammer5!

**Pull request merged**